### PR TITLE
update solr

### PR DESCRIPTION
**Pull request by whereismyjetpack**
- updates solr in docker to match what's in qa

### CODE REVIEW

**Comment by whereismyjetpack** [comment]
@nicktiberi I finally found a way to get this to work

File: docker-compose.yml [hunk 1]
```diff
@@ -81,18 +82,20 @@ services:
     - -c 
     - mkdir -p /data/scholarsphere; minio --compat server --address ':9000' /data
   solr:
-    image: psul/solr:8.5.2-slim
+    image: solr:8.11.1-slim
     restart: always
     environment:
-      SOLR_STOP_WAIT: 5
+      SOLR_STOP_WAIT: 10
```
Comment (nicktiberi) [inline docker-compose.yml:88]: Just for my own info, what is the significance of this environment variable and why did it change?

Comment (whereismyjetpack) [reply docker-compose.yml:88]: SOLR_STOP_WAIT is the time solr will wait before forcefully killing it. it defaults to something crazy like 180. as it turns out that variable is also used for the start_wait time, and i had to increase it.

**Review by nicktiberi — APPROVED** [review]
LGTM! Something seems to be weird with Codeclimate, but this change wouldn't affect any of those results.

**Comment by whereismyjetpack** [comment]
I'll put in a ticket to look into code climate

**Pull request merged**